src/lib/skilling/skills/crafting/craftables/misc.ts

```diff
@@ -88,6 +88,46 @@ const Misc: Craftable[] = [
 		inputItems: new Bank({ 'Xerician fabric': 5 }),
 		tickRate: 3
 	},
+	{
+		name: 'Splitbark gauntlets',
+		id: itemID('Splitbark gauntlets'),
+		level: 60,
+		xp: 62,
+		inputItems: new Bank({ Bark: 1, 'Fine cloth': 1 }),
```

Comment:
From now forward, we no longer want to use this format of passing in names to the constructor, because we want to eventually get rid of it. instead, do: `new Bank().add().add()`